### PR TITLE
docs: note deprecated status of senna.js repo

### PR DESCRIPTION
**Pull request by wincent**
As per these tickets:

- https://issues.liferay.com/browse/LPS-122528
- https://issues.liferay.com/browse/LPS-122534

### CODE REVIEW

**Comment by wincent** [comment]
`3.x` version of this change is: https://github.com/liferay/senna.js/pull/346

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+> :warning: In the future, [the `senna` npm package](https://npmjs.com/package/senna) will be deprecated and this repo archived (ie. switched to read-only mode). Development continues in [the `liferay/liferay-portal` repo](https://github.com/liferay/liferay-portal) (more specifically, inside [the `frontend-js-spa-web` module](https://github.com/liferay/liferay-portal/tree/master/modules/apps/frontend-js/frontend-js-spa-web)), but not as an independent project, in order to better integrate with and serve the needs of Liferay DXP.
+
+---
```
Comment (jbalsas) [inline README.md:3]: Let's add also something like...

> The `2.x` version is the last official release blabla... if you're interested in becoming a maintainer to improve this let us know or for it or I don't know, something!!

#justsaying

Comment (wincent) [reply README.md:3]: anyone

One thing to bear in mind is that from a legal perspective, we open ourselves up to liability by allowing a third-party to take ownership (or even just write privileges) on a package that is (or was) associated with the Liferay brand. People may have `senna` in their package.json files with a lose enough version range that they'll unwittingly get a copy of the library from an entity other than the one they initially chose to trust...

`#justsaying`

Comment (jbalsas) [reply README.md:3]: Fair enough, let's then simply state that 

> `2.x` is the last official release version that will be used only for maintenance as long as DXP versions that use it haven't reached EOL. 

If anyone is really interested in picking this up, they can let us know and we can figure out what that really means.

How does that sound?

**Comment by wincent** [comment]
Assuming this is ok now after the edits, going to _moige_ it.

**Pull request merged**